databaseconnect.py

```diff
@@ -250,7 +250,7 @@ def clear_table(table_name):
 
         if input("Enter 'Y' to confirm cleaning of BOTH tables: ") in ("Y", "y",):
             for table in tables_to_be_cleaned:
-                cur.execute("DELETE FROM %s", (table,))
+                cur.execute("""DELETE FROM %s""" % table)
```

Comment:
Can you clarify why `cur.execute("""DELETE FROM %s""" % table)` is being used instead of cur.execute("DELETE FROM %s" % table)`?

Reply:
Actually, I followed the documentation and used docstrings ("""). But anyway works fine.

Reply:
While I was trying to change them to quotes, I encountered check fails. Using any of them is fine.

Reply:
Since we see `cur.execute` statements in this code mostly using `"` instead of `"""`, it'd be better of you could make the change (for the sake of consistency). :sweat_smile:

Reply:
Yeah, I am sorry for not being active, I have changed it. Just trying to rectify the checks.